helmfile/values/prometheus-alerts-sc.yaml.gotmpl

```
@@ -1,8 +1,8 @@
-esNodeCount: {{ add .Values.elasticsearch.dataNode.count .Values.elasticsearch.clientNode.count .Values.elasticsearch.masterNode.count }}
-eskubeAuditSizeGB: {{ .Values.elasticsearch.curator.retention.kubeAuditSizeGB }}
-eskubernetesSizeGB: {{ .Values.elasticsearch.curator.retention.kubernetesSizeGB }}
-esotherSizeGB: {{ .Values.elasticsearch.curator.retention.otherSizeGB }}
-esauthLogSizeGB: {{ .Values.elasticsearch.curator.retention.authLogSizeGB }}
+esNodeCount: {{ add .Values.opensearch.dataNode.count .Values.opensearch.clientNode.count .Values.opensearch.masterNode.count }}
+eskubeAuditSizeGB: {{ .Values.opensearch.curator.retention.kubeAuditSizeGB }}
+eskubernetesSizeGB: {{ .Values.opensearch.curator.retention.kubernetesSizeGB }}
+esotherSizeGB: {{ .Values.opensearch.curator.retention.otherSizeGB }}
+esauthLogSizeGB: {{ .Values.opensearch.curator.retention.authLogSizeGB }}
```

Use `os` as shot notation for opensearch?
```suggestion
osNodeCount: {{ add .Values.opensearch.dataNode.count .Values.opensearch.clientNode.count .Values.opensearch.masterNode.count }}
osKubeAuditSizeGB: {{ .Values.opensearch.curator.retention.kubeAuditSizeGB }}
osKubernetesSizeGB: {{ .Values.opensearch.curator.retention.kubernetesSizeGB }}
osOtherSizeGB: {{ .Values.opensearch.curator.retention.otherSizeGB }}
osAuthLogSizeGB: {{ .Values.opensearch.curator.retention.authLogSizeGB }}
```